sinonjs/samsam
# Remove superfluous prettier config

This file seems to be a leftover. I verified that wrong indentation still produces a linter error.


#### How to verify

1. Check out this branch
2. `npm ci`
3. `npm run lint` passes
4. Change indendation somewhere to tab width 2
5. `npm run lint` fails
